packages/discussions/graphql/resolvers/Comment.js

```diff
@@ -191,13 +205,11 @@ module.exports = {
       }
       : {
         id,
-        name: commenter.name || t('api/noname'),
+        name: name || t('api/noname'),
         profilePicture: profilePicture,
         credential,
         anonymity: false,
-        username: commenter._raw.hasPublicProfile
-          ? commenter._raw.username
-          : null
+        username
```

Comment:
let's expose both fields on `DisplayUser`: `slug` (add to schema) & `username` (deprecate & already use new slug logic)

Reply:
fyi: the code was already updated to use the new slug resolver of `User` by the time of your comment. I still exposed now both `username` and `slug` on `DisplayUser`. Is it now how you imagined it?

Reply:
@patte perfect